lib/image.js

```diff
@@ -1,12 +1,84 @@
-"use strict";
-var Promise = require('bluebird');
-var fs = require('fs-extra');
+'use strict';
+var Cesium = require('cesium');
+var fsExtra = require('fs-extra');
 var path = require('path');
+var PNG = require('pngjs').PNG;
+var Promise = require('bluebird');
+
+var fsExtraReadFile = Promise.promisify(fsExtra.readFile);
 
-var fsReadFile = Promise.promisify(fs.readFile);
+var defaultValue = Cesium.defaultValue;
+var WebGLConstants = Cesium.WebGLConstants;
 
 module.exports = loadImage;
```

Comment:
This is a problem with most of the files in `lib`.  They should all be fixed.  We can do that in a quick separate PR after this is merged if you want, but please write up an issue.